APP-DEPLOYMENT.md

```
@@ -0,0 +1,722 @@
+
+### Image Requirements
+
+**All container images MUST support both architectures** (`linux/arm64` and `linux/amd64`).
+
+**Verify multi-arch support**:
+```bash
+docker buildx imagetools inspect <image>:<tag> | grep Platform
+```
+
+Expected output includes both:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)